bundle/tests/scorecard/disabled/sso1-social/00-secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  # Name of the secret should be in this format: <OpenLibertyApplication_name>-olapp-sso
+  name: sso-wsliberty-olapp-sso
```

Comment:
This should be in the format <WebSphereLibertyApplication_name>-wlapp-sso

Since the sso tests are disabled. It's fine to merge this. Please change as part of the PR to enable tests.